rbi/gems/test_case.rbi

```diff
@@ -0,0 +1,12 @@
+# typed: __STDLIB_INTERNAL
+
+class ActiveSupport::TestCase < ::Minitest::Test
+  sig { params(args: T.untyped, block: T.nilable(T.proc.bind(T.attached_class).void)).void }
+  def self.setup(*args, &block); end
```

Comment:
Your PR description called out qualms with the fact that these methods are not actually defined in real life, that they're defined in some module.

That's another point in favor of having these RBIs only in the test cases and projects that want this behavior, not in Sorbet.